tests/console/clamav.pm

```diff
@@ -40,6 +40,12 @@ sub run {
     $self->select_serial_terminal;
 
     zypper_call('in clamav vim');
+
+    # Check Clamav version
+    # Jira ticket : SLE-16780 : upgrade Clamav SLE
+    zypper_call('info clamav');
```

Comment:
Why there are " " (blank) after "ticket" and "16780".
Is it ok not check the expected version?

Reply:
I will remove the blank. :) Check the expected version is fine, but in the Jira ticket, we still do not reliaze the version will be released finally. So I do not check the fixed version here.